src/quo/theme.cljs

```diff
@@ -42,16 +42,15 @@
 (defn use-theme
   "A hook that returns the current theme context."
   []
-  (utils.transforms/js->clj (rn/use-context theme-context)))
+  (keyword (rn/use-context theme-context)))
```

Comment:
btw we had initially made this return a map so that the api is easier to extend if we ever need to.
`use-theme-value` would get the result directly and you could just use that here.